rllib/connectors/connector_v2.py

```diff
@@ -149,8 +127,232 @@ def __call__(
             kwargs: Forward API-compatibility kwargs.
 
         Returns:
-            The transformed connector output abiding to `self.output_type`.
+            The transformed connector output.
+        """
+
+    @staticmethod
```

Comment:
New convenience API for unifying multi-agent and single-agent cases (at least if multi-agent means we are simply looping through the individual single-agent episodes within the multi-agent ones).